### PR TITLE
Feature/checkbirthdate

### PR DESCRIPTION
**Pull request by Cardamced**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [X] Frontend
- [X] Backend
- [ ] Other: .....

## General checklist:

- [X] My code follows the style guidelines of this project
- [X] I ran `bun run format`/`bun run lint` before commit
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation where needed
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] I have updated `develop` and merged to my branch before submitting pull request
- [X] My pull request generate no conflicts with `develop` branch
- [X] I requested code review from other team members

## Frontend checklist:

- [X] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [X] My `Javascript` generate no new console errors
- [X] I tested my code cross browsers
- [X] My slice is pixel perfect for both desktop and mobile according to design
- [X] I conducted basic QA to assure all features are working
- [X] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [X] I added description to admin fields in hard-to-understand areas
- [X] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by No3ml3 — DISMISSED** [review]

**Review by jaalazard — DISMISSED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -16,5 +16,6 @@
     "forceConsistentCasingInFileNames": true,
     "allowJs": true,
     "types": ["bun-types"]
-  }
+  },
+  "extends": "expo/tsconfig.base"
```
Comment (YDrogen) [inline tsconfig.json]: nope

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: tsconfig.json (hunk 1 above)
Comment (Cardamced) [reply tsconfig.json]: Done.

Comment (YDrogen) [reply tsconfig.json]: Non

Comment (Cardamced) [reply tsconfig.json]: J'aurions cru. Dingue. Maintenant c'est bon.

Comment (YDrogen) [reply tsconfig.json]: Toujours pas :(

Comment (Cardamced) [reply tsconfig.json]: Running gag. :'-/

**Review by YDrogen — APPROVED** [review]

**Pull request merged**